routes/target.js

```diff
@@ -0,0 +1,37 @@
+var express = require("express");
+var router = express.Router();
+const db = require("../model/helper");
+
+// Postman Test = OK http://localhost:5000/api/target/test
+router.get("/test", (req, res) => {
+    res.send("Test route is working!");
+  });
+
+// Postman Test = OK http://localhost:5000/api/target/targets
+router.get("/targets", async (req, res) => {
+    console.log('GET /api/target/targets route hit');
+    try {
+      // Search the database to retrieve all the targets
+      const result = await db("SELECT * FROM target");
+      console.log('Query result:', result); // Log the result for debugging
+      
+      // Check if there's an error and handle it
+      if (result.error) {
```

Comment:
Remove this is as it's not necessary. If an error occurs in an async function, it will typically throw an exception. You should handle errors within the catch block instead of checking manually in the try block.

```
router.get("/targets", async (req, res) => {
  try {
    const result = await db("SELECT * FROM targets");
    res.json(result.data);
  } catch (error) {
    console.error("Error retrieving targets", error);
    res.status(500).json({ message: "Database query failed" });
  }
```